spec/requests/submissions_spec.rb

```diff
@@ -8,17 +8,18 @@
   let(:student) { create(:student) }
   let(:course) { create(:course) }
```

Comment:
Instead of appending to the `course.students` before each test case, you can initialize the course with `create(:course, students: [student])`